app/DailyMenu.php

```diff
@@ -7,15 +7,30 @@
 
 class DailyMenu extends Model
 {
-    use softDeletes;
+    use SoftDeletes;
 
+    protected $timestamp = true;
+    protected $dates = ['deleted_at'];
     /**
-     * DailyMenu has many Food
+     * The table associated with the model.
      *
-     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     * @var string
      */
-    public function foods()
+    protected $table = 'daily_menus';
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = ['quantity', 'created_at', 'updated_at'];
+
+    /**
+     * Get the food for the menu item.
+     *
+     * @return relationship
+     */
+    public function food()
```

Comment:
I'm sorry, I don't have a clear feedback. You should return type relationship. example:
@return \Illuminate\Database\Eloquent\Relations\HasMany.